web/styles/app_components.css

```diff
@@ -842,3 +842,15 @@ div.overlay {
         font-size: 1em;
     }
 }
+
+#settings_content,
+#stream_settings,
+#edit_bot_modal {
+    .dropdown-list-widget .dropdown-list-wrapper {
+        width: fit-content;
+
+        span {
+            width: fit-content;
+        }
```

Comment:
Added this here because we do not want these changes for dropdown-list-widget in "Move topic" modal.

Reply:
Hmm, what happens if we have these on all dropdown-list-widgets?

I'm wondering if we can express this idea through some sort of option or configuration.

Probably fine to do in #frontend though -- not a blocker for this PR.

Reply:
Adding this to all widgets, leads to issue in the "Move topic" menu as we have set the postiion of the menu to be `fixed` for that case for it to be completely visible and that results in unexpected UI with the submit buttons appearing on top of the dropdown menu.

Can try adding it as an config option though.